blockchains/smartchain/assets/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e/info.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Veraswap Protocol",
+    "website": "https://veraswap.org",
+    "source_code": "https://github.com/Virginlink",
+    "description": "Fully Dex protocol, staking and more on Binance Smart Chain. #Defi #BSC",
+    "socials": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/veraswap",
+            "handle": "Veraswap Protocol"
+        },
+        {
+            "name": "Telegram",
+            "url": "https://t.me/veraswap",
+            "handle": "Veraswap Protocol"
+        }
+    ],
+    "explorer": "https://bscscan.com/token/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e",
+    "type": "BEP20",
+    "symbol": "VRAP",
+    "decimals": 18,
+    "status": "active",
+    "id": "0x271c418b045d05a1d52c6bf849d47b5b5b4d769e"
```

Comment:
`0x271c418b045d05a1d52c6bf849d47b5b5b4d769e` should be checksummed version for info and file name: `0x271C418B045d05A1D52c6bF849d47b5B5B4d769e`

Reply:
https://github.com/trustwallet/assets/pull/5906

I've created a new pull request with the correct info as I cannot edit the current request

Reply:
Pleae check the above commit